tedbottompicker/src/main/java/gun0912/tedbottompicker/CameraPermissionActivity.java

```diff
@@ -38,9 +38,22 @@
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        RxPermissions rxPermissions = new RxPermissions(this);
+        if (savedInstanceState != null) {
```

Comment:
[not reproducable] sometimes this activty gets restarted if the camera activity starts => lead to infinite camera loops